gammapy/maps/region/ndmap.py

```diff
@@ -243,48 +247,52 @@ def create(
         Parameters
         ----------
         region : str or `~regions.SkyRegion`
-            Region specification
-        axes : list of `MapAxis`
-            Non spatial axes.
-        dtype : str
-            Data type, default is 'float32'
-        unit : str or `~astropy.units.Unit`
-            Data unit.
-        meta : `dict`
-            Dictionary to store meta data.
-        wcs : `~astropy.wcs.WCS`
-            WCS projection to use for local projections of the region
-        binsz_wcs: `~astropy.units.Quantity` or str
+            Region specification.
+        axes : list of `MapAxis`, optional
+            Non-spatial axes. Default is None.
+        dtype : str, optional
+            Data type. Default is 'float32'.
+        unit : str or `~astropy.units.Unit`, optional
+            Data unit. Default is "".
+        meta : `dict`, optional
+            Dictionary to store metadata.
+            Default is None.
+        wcs : `~astropy.wcs.WCS`, optional
+            WCS projection to use for local projections of the region.
+            Default is None.
+        binsz_wcs: `~astropy.units.Quantity` or str, optional
             Bin size used for the default WCS, if wcs=None.
-        data : `~numpy.ndarray`
-            Data array
+            Default is "0.1deg".
```

Comment:
```suggestion
            Default is "0.1 deg".
```

Reply:
The default value is actually set to "0.1deg", is that a mistake then?